docs/eventing/samples/apache-camel-source/display_resources.yaml

```diff
@@ -1,14 +1,13 @@
 # Channel for testing events.
 
-apiVersion: eventing.knative.dev/v1alpha1
+apiVersion: messaging.knative.dev/v1alpha1
```

Comment:
Same here, and other places where you create the channel directly. Just use the kind: InMemoryChannel.